src/lang/en.js

```diff
@@ -191,6 +203,29 @@ const locale = {
         whenRatioReaches: 'When ratio reaches',
         whenSeedingTimeReaches: 'When seeding time reaches'
       },
+      pageRss: {
+        tabName: {
+          general: 'Geral',
```

Comment:
this seems like a typo :p

Reply:
Also, I think you need to add 'feed' at the top here in the 'General' section... at the top of the file

Reply:
A mix of english and portuguese hahaha Will fix it